src/BenchmarksServer/Startup.cs

```diff
@@ -211,28 +211,12 @@ private static string CloneAndRestore(string path, ServerJob job)
 
             Debug.Assert(benchmarksDir != null);
 
-            // Modify all global.json to reference source dirs
```

Comment:
Could we use @pakrym's powershell script to modify the csproj's package references to make them package references?